test/AcquiaSpecTest.php

```diff
@@ -14,7 +14,7 @@
 use GuzzleHttp\Psr7\Request;
 use GuzzleHttp\Psr7\Response;
 
-class AcquiaSpecTest extends \PHPUnit_Framework_TestCase
+class AcquiaSpecTest extends \PHPUnit\Framework\TestCase
```

Comment:
Let's move these to `use` statements since they are no longer in the top-level namespace.